claat/render/template.go

```diff
@@ -27,11 +27,12 @@ import (
 
 // Context is a template context during execution.
 type Context struct {
-	Env      string
-	Prefix   string
-	GlobalGA string
-	Meta     *types.Meta
-	Steps    []*types.Step
+	Env       string
+	Prefix    string
+	GlobalGA  string
+	Meta      *types.Meta
+	Steps     []*types.Step
+	ExtraVars map[string]string // Extra variables passed from the command line.
```

Comment:
Maybe just name it `Extra`? It's shorter :) and still makes sense to what it is, I think.